tests/tests_wireless_nm.yml

```diff
@@ -0,0 +1,17 @@
+---
+# set network provider and gather facts
+- hosts: all
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+
+# workaround for: https://github.com/ansible/ansible/issues/27973
+# There is no way in Ansible to abort a playbook hosts with specific OS
+# releases Therefore we include the playbook with the tests only if the hosts
+# would support it.
+# mac80211_hwsim only seems to be available and working on RHEL/CentOS 7
```

Comment:
Just clarify, it is also not available on Fedora?

Reply:
Fedora has it available in the kernel-modules-internal rpm, but it crashed every time i tired to load it on Fedora31